.evergreen/evergreen.yml.in

```diff
@@ -1204,34 +1206,26 @@ buildvariants:
     tasks:
       - name: e2e_tests_linux_x64
       - name: e2e_tests_linux_x64_openssl11
-  - name: e2e_amazon1_x64
-    display_name: "Amazon Linux 1 x64 (E2E Tests)"
-    run_on: amazon1-2018-small
-    tasks:
-      - name: e2e_tests_linux_x64_50x
   - name: e2e_amazon2_x64
     display_name: "Amazon Linux 2 x64 (E2E Tests)"
-    run_on: amazon2-small
+    run_on: amazon2-large
     tasks:
       - name: e2e_tests_linux_x64
   - name: e2e_amazon2023_x64
     display_name: "Amazon Linux 2023 x64 (E2E Tests)"
     run_on: amazon2023.0-small
+    expansions:
+      disable_openssl_shared_config_for_bundled_openssl: true
     tasks:
       - name: e2e_tests_linux_x64_700rc10
-  - name: e2e_suse12_x64
-    display_name: "SLES 12 x64 (E2E Tests)"
-    run_on: suse12-sp5-small
-    tasks:
-      - name: e2e_tests_linux_x64
```

Comment:
Calling this out - let me know if removing this is fine otherwise I will put it back.

Reply:
Mh – was there a specific reason? I think this would require a scope update, we can do that if there’s a specific reason but otherwise we should keep this test

Reply:
For some reason I recalled having a conversation about it where we decided that it would be fine testing suse only on 15 (most likely because of older glibc on 12) but that is probably just my imagination because I just went on the scope again it does mention that we will continue testing on 12. Sorry for the confusion, I will revert this removal.